usr/src/uts/common/fs/zfs/dsl_scan.c

```diff
@@ -1446,6 +1446,10 @@ dsl_scan_active(dsl_scan_t *scn)
 	return (used != 0);
 }
 
+/*
+ * Called whenever a txg syncs.  Called for both current txgs and old ones that
+ * are being resilvered
```

Comment:
Hm, it's been a long time since I wrote this, but I'm guessing that I might have been confused by incorrect dtrace usage?  I guess I'll just delete that sentence.